docs/extend/config.md

```diff
@@ -171,7 +170,6 @@ Config provides the base accessible fields for working with V0 plugin format
 
 ```
 {
-       	"configVersion": "v0",
```

Comment:
It's a doc left-over from a previous PR, it's already removed from the code, after talking with @tiborvass and @stevvooe. 

* You needed to json-decode in order to get the version, if the format change a lot, it won't even work.
* The version is already present in the media-type, that's how other media handle versioning.